community/louvain.tex

```diff
@@ -190,97 +190,278 @@ \section{Directed Graphs}
 
 It is of interest to consider how this generalises to directed graphs.
 If we are to treat incoming and outgoing nodes equally, there are several
-thoughts on how to extend the algorithm to directed graphs: 
+thoughts on how to extend the algorithm to directed graphs, of which we 
+will explore three: 
 \begin{itemize}
-	\item to generalise the expressions to the directed case -- there are
-	two obvious ways one could go about this. 
-	\item to construct an undirected graph by averaging the weights of edges 
-	in opposite directions, and then use the undirected case.
+	\item Construct an undirected graph first, and then use the undirected case.
+	\item Generalise the expressions from the undirected case to the directed case, 
+	we will consider two different suggestions for such generalisations. 
 \end{itemize}
-We will show that two of these three approaches are equivalent, and discuss
-the differences to the third.
-
-\subsection{First generalisation of expressions approach}
-One suggested extension to directed graphs is to modify the expressions 
-involved by adding the `from' case and the `to' case for each term. 
-If we let $B_{ij}$ be the edge weight between nodes $i$ and $j$ in 
-the directed graph of interest, these extended expressions look like
-\begin{equation}
-m^{(1)} = \frac{1}{2}\left ( \sum_i\sum_jB_{ij} + \sum_i\sum_jB_{ji}\right)  = \frac{1}{2}\sum_i\sum_j \left( B_{ij} + B_{ji} \right) ,
-\end{equation}
-\begin{equation}
-k_i^{(1)} = \sum_jB_{ij} + \sum_jB_{ji} = \sum_j{\left( B_{ij} + B_{ji} \right)},
-\end{equation}
-and similarly
-\begin{equation}
-	Q^{(1)} = \frac{1}{2m}\sum_i\sum_j\left[ \left( B_{ij} + B_{ji} \right) - \gamma \frac{k_ik_j}{2m} \right] \delta(c_i,c_j).
-\end{equation}
+We will show that one of the two `generalisation of expressions' approachs is 
```

Comment:
Done. Such a minor change -- and this diff isn't very informative anyway, should I just squash it straight away?